services/src/main/java/org/exoplatform/news/service/impl/NewsServiceImplV2.java

```diff
@@ -167,4 +228,91 @@ public void unarchiveNews(String newsId, String currentUserName) throws Exceptio
+      // created and updated date set by default during the draft creation process
+      draftArticle = noteService.createDraftForNewPage(draftArticle, System.currentTimeMillis());
+      if (draftArticle != null) {
+        // save illustration
+        Long illustrationId = -1L;
+        if (news.getUploadId() != null && !news.getUploadId().isBlank()) {
+          illustrationId = saveNewsIllustration(news.getUploadId(), null);
+        }
+        Map<String, String> properties = new HashMap<>();
+        properties.put("summary", news.getSummary());
+        properties.put("publicationState", news.getPublicationState());
+        properties.put("uploadId", news.getUploadId());
+        properties.put("illustrationId", illustrationId >= 0 ? String.valueOf(illustrationId) : null);
+        NewsDraftObject newsDraftMetaDataObject =
+                                                new NewsDraftObject(NEWS_METADATA_DRAFT_OBJECT_TYPE, draftArticle.getId(), null);
+        MetadataKey newsDraftMetadataKey = new MetadataKey(METADATA_TYPE.getName(), "news", 0);
+        metadataService.createMetadataItem(newsDraftMetaDataObject, newsDraftMetadataKey, properties);
+        news.setId(draftArticle.getId());
+        news.setCreationDate(draftArticle.getCreatedDate());
+        news.setUpdateDate(draftArticle.getUpdatedDate());
+        if (illustrationId !=null) {
+          FileItem fileItem = fileService.getFile(illustrationId);
+          if (fileItem != null) {
+            news.setIllustration(fileItem.getAsByte());
+          }
+        }
+        return news;
+      }
+    }
+    throw new EntityNotFoundException("Parent note not found");
+  }
+
+  protected Page getArticlesRootPage(String ownerId) {
+    List<Page> notes =
+                     noteService.getNotesOfWiki("group", ownerId)
+                                .stream()
+                                .filter(e -> e.getName().equals(NEWS_ARTICLES_ROOT_NOTE_PAGE_NAME) && e.getParentPageId() == null)
+                                .toList();
+    return notes.isEmpty() ? null : notes.get(0);
+  }
+
+  private Long saveNewsIllustration(String uploadId, Long oldFileId) {
+    if (uploadId == null || uploadId.isBlank()) {
+      throw new IllegalArgumentException("uploadId is mandatory");
+    }
+    if (oldFileId != null && oldFileId != 0) {
+      fileService.deleteFile(oldFileId);
+    }
+    UploadResource uploadResource = uploadService.getUploadResource(uploadId);
+    if (uploadResource == null) {
+      throw new IllegalStateException("Can't find uploaded resource with id : " + uploadId);
+    }
+    try {
+      InputStream inputStream = new FileInputStream(uploadResource.getStoreLocation());
+      FileItem fileItem = new FileItem(null,
+                                       uploadResource.getFileName(),
+                                       uploadResource.getMimeType(),
+                                       FILE_API_NAME_SPACE,
```

Comment:
```suggestion
                                       FILE_API_NAME_SPACE,
```
define a namespace for news